### ISSUE TITLE
Inconsistent bindings for `hint links yank[-primary]`

### ISSUE THREAD

**Issue by Ambrevar** [qutebrowser/qutebrowser#2363]
The default bindings follow this rule
- lowercase for primary selection;
- uppercase for clipboard.

Except for this one:
```
hint links yank
    ;y

hint links yank-primary
    ;Y
```

Is there a rationale behind this? I find it confusing.

**Comment by The-Compiler** [comment]
They don't - `yy` yanks to clipboard, `yY` to primary.

**Comment by Ambrevar** [comment]
Right, from a fresh basedir I've got a consistent behaviour. From my config that was updated since early versions of Qutebrowser, it's not. I suppose the automatic config updater messed up at some point since I've never touched those settings.
